otter/integration/tests/test_convergence.py

```diff
@@ -827,7 +856,134 @@ def wrapper(self, *args, **kwargs):
                 "See #881.")
         return (name, wrapper)
 
+    @skip_me("Autoscale does not support error status yet. See #885")
+    @tag("CATC-020")
+    def test_delete_all_loadbalancers_and_scale_up(self, delete_clb=None):
+        """
+        CATC-020-a. This will also tested with starting with 2 load balancers.
+
+        1. Creates a scaling group with a load balancer and 1 server.
+        2. Ensure that the server is active and added to the load balancer.
+        3. Delete the load balancer using the delete command (it may set
+           the load balancer to PENDING_DELETE instead, for example)
+        4. Attempt to scale up by 1.
+        5. Assert that the scaling group goes into error state, that there are
+           no more active servers added to the group, but the previous active
+           server is not deleted.
+
+        :param callable delete_clb: function that takes a test resource
+            and a load balancer and deletes (or pending-deletes) the
+            load balancer.
+        """
+        group = self.helper.create_group(
+            image_ref=image_ref, flavor_ref=flavor_ref, min_entities=1)
+
+        return (
+            self.helper.start_group_and_wait(group, self.rcs)
+            .addCallback(_delete_a_clb_and_scale, self.helper, group,
+                         scale_by=1, delete_clb=delete_clb)
+            .addCallback(
+                group.wait_for_state,
+                MatchesAll(
+                    ContainsDict({
+                        'pendingCapacity': Equals(1),
```

Comment:
@radix: Looks like my assumption about this was wrong - active goes to zero, and pending becomes 2 again, because since the original server is not on any load balancer anymore.  Is that correct?  Is it weird that if the load balancer goes away, the group's active servers go to zero, but we don't delete the servers from nova?